### PR TITLE
Add L'math 1.6

### PR DESCRIPTION
**Pull request by TheJiahao**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate <manifest>`, where `<manifest>` is the name of the manifest you're submitting?
- [x] Have you tested your manifest locally with `winget install -m <manifest>`?

-----

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/winget-pkgs/pull/2366)

### CODE REVIEW

File: manifests/RoniLehto/Lmath/1.6.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+InstallerType: INNO
+LicenseUrl: https://www.lehtodigital.fi/lmath/?p=license
+AppMoniker: LMath
+Tags: math, latex, editor
+Description: L'Math is an Electron powered text editor that is mainly used by Finnish students preparing for their digital matriculation exams. L'Math has the same LaTeX editor built-in that is found on the Digabi platform.
+Homepage: https://www.lehtodigital.fi/lmath/
+Installers:
+  - Arch: x86
+    InstallerType: EXE
+    Url: https://www.lehtodigital.fi/lmath/proxy.php?download=win32
```
Comment (OfficialEsco) [inline manifests/RoniLehto/Lmath/1.6.yaml:15]:
```suggestion
    Url: https://github.com/lehtoroni/lmath-issues/releases/download/r1.6/LMath_Windows_r1.6.exe
```
`https://www.lehtodigital.fi/lmath/proxy.php?download=win32` redirects to `https://github.com/lehtoroni/lmath-issues/releases/download/r1.6/LMath_Windows_r1.6.exe`

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Redirect fix, also sent to smartscreen just incase

**Comment by TheJiahao** [comment]
@OfficialEsco The suggestion sounds reasonable, committed. Additionally, the bot removed the labels as well.

**Review by ghost — APPROVED** [review]
Validation has completed

**Pull request merged**